app/views/email_authentications/passwords/edit.html.erb

```diff
@@ -16,3 +16,7 @@
 <div class="actions mt-4">
   <%= render "email_authentications/shared/links" %>
 </div>
+
+<script>
+  new PasswordVisibilityToggle
```

Comment:
minor thing, but can you throw parens at the end of this function call?

```
new PasswordVisibilityToggle();
```

Reply:
Good call. Thanks. Something felt off about that. Most of my JS experience is in frameworks, so I sometimes feel out of my element in vanilla JS.